addons/medical_treatment/functions/fnc_placeInBodyBagOrGrave.sqf

```diff
@@ -0,0 +1,58 @@
+#include "script_component.hpp"
+/*
+ * Author: Glowbal, drofseh
+ * Places a dead body inside a body bag.
+ *
+ * Arguments:
+ * 0: Arguments <ARRAY>
+ *   0: Medic <OBJECT>
```

Comment:
```suggestion
 *   0.0: Medic <OBJECT>
```

Reply:
0.0 style is only used in a couple ACE functions, the indented style I used is much more common (2 vs 19)